circle.yml

```diff
@@ -0,0 +1,82 @@
+version: 2
+workflows:
+  version: 2
+  linux-builds:
+    jobs:
+      - build-x86_64
+      - build-i686
+
+jobs:
+  build-x86_64:
+    docker:
+#      - image: ubuntu:14.04
+      - image: circleci/python:2.7
+        environment:
+          JULIA_CPU_CORES: 4
```

Comment:
I did a bunch of test builds on my fork - running `nproc` in the v2 container says 32 which is probably more to do with the container host server, if you run without a JULIA_CPU_CORES limit then things get killed. pretty sure I tested this out set to different values from 2-6 and 4 was fastest overall, but can try more of those. it isn't worth manually load balancing across separate containers, the concurrency of multiple builds is better spent on testing multiple pr's simultaneously for overall throughput